packages/ng/forms/multilanguage-input/multilanguage-input.component.html

```diff
@@ -27,7 +27,11 @@
 				<div class="popover-contentOptional multilanguage_popover-content" [style.width.px]="inputElement.clientWidth + 16">
 					<div class="u-displayFlex pr-u-gap100 u-flexDirectionColumn">
 						@for (row of panelInputs(); track row.cultureCode) {
-						<lu-form-field [label]="intl.translateTo | intlParams: {lang: getLocaleDisplayName(row.cultureCode)}" hiddenLabel>
+						<lu-form-field
+							[size]="formFieldSize"
```

Comment:
With the other suggestion, this needs to be a signal call.

```suggestion
							[size]="formFieldSize()"
```